imageio/plugins/rawpy.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# imageio is distributed under the terms of the (new) BSD License.
```

Comment:
We typically don't add a license header. Any reason you've added it here?

Reply:
I just saw it in other file and assumed it was needed. I have removed it now.